src/components/Repository.js

```diff
@@ -125,10 +163,15 @@ function Repository({match}) {
               <a rel="noreferrer" target="_blank" href={`https://codetriage.com/${nameWithOwner}`}>
                 <Button primary>Set up CodeTriage</Button>
               </a>
+              {isForked ?
+                <a rel="noreferrer" target="_blank" href={`https://github.com/${user.login}/${repoName}`}>
+                  <Button disabled={isForkLoading} data-test="go-to-fork-button">Go to fork</Button>
+                </a> :
+                <Button disabled={isForkLoading} onClick={forkRepository}>Fork {humanizeNumber(forks.totalCount)}</Button>}
```

Comment:
Can we leverage Octicon to the button @MKrupauskas? What do you think?

Reply:
Which icons would fit for which buttons?

Reply:
ForkIcon for fork button!